### ISSUE TITLE
How does autocontainmode "relaxed" work?

### ISSUE THREAD

**Issue by hejops** [tridactyl/tridactyl#3494]
Tridactyl version: 1.21.0pre5424-85d7bcbe
Firefox version: Mozilla Firefox 86.0.1
Operating system: linux

-   Brief description of the problem:
`relaxed` mode doesn't seem to assign tabs to the expected container, neither when opened as a new tab nor from an existing tab. Is this behaviour normal?

-   Steps to reproduce:

```
set auconcreatecontainer true
set autocontainmode relaxed
autocontain -s archlinux.org tech

tabopen archlinux.org
open archlinux.org
```

**Comment by bovine3dom** [comment]
Yeah, it must be broken, thanks for filing the issue! `tabopen archlinux.org` is supposed to be in the `tech` container in your example.

Probably introduced by ea257f364 or 85129c346, something like that.

I'm a bit perplexed though because I think https://github.com/tridactyl/tridactyl/blob/ee8bd48d46c1550282d89ec066d9a1808908fcba/src/excmds.ts#L2392 should still work : /
